examples/jenkins/README.md

```diff
@@ -10,46 +9,76 @@ the deployment works, and then tag the test version into production.
 Steps
 -----
 
-1. Follow steps 1-10 from the [sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md)
+1. Follow steps 1-7 from the [sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md)
 
-2. Start the Jenkins services
+2. Add the `edit` role to the `default` service account:
 
-        $ oc create -n test -f jenkins-config.json
-        
-3. Create the application configuration
+        $ oc policy add-role-to-user edit system:serviceaccount:test:default
 
-        $ oc process -n test -f application-template.json | oc create -n test -f -
- 
-4. Locate the Jenkins service endpoint and go to it in your browser:
+    This will allow the service account Jenkins uses to run builds and tag images.
 
-        $ oc get services -n test jenkins --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}"
+3. Identify the service account token secret:
```

Comment:
If so, it might be better to make these optional steps at the end - "If you want Jenkins to run against a different cluster, enter the API URL and a token by doing ...."